app/views/organisations/not_live.html.erb

```diff
@@ -22,7 +22,7 @@
     <div class="inner-block">
       <div class="description">
         <%= govspeak_to_html @organisation.summary %>
-        <% unless @organisation.closed? %>
+        <% unless @organisation.closed? || @organisation.court_or_hmcts_tribunal? %>
```

Comment:
"Not Live" doesn't necessarily mean "Closed". It could be exempt or transitioning. Closed means was a thing but is no longer a thing. 
So the logic is: If the org is closed or is a court or tribunal, don't show this thing.